test/e2e/volumes.go

```diff
@@ -76,7 +75,6 @@ func DeleteCinderVolume(name string) error {
 	return err
 }
 
-
 // These tests need privileged containers, which are disabled by default.  Run
 // the test with "go run hack/e2e.go ... --ginkgo.focus=[Feature:Volumes]"
 var _ = framework.KubeDescribe("Volumes [Feature:Volumes]", func() {
```

Comment:
The `[Feature:]` tag was present when I introduced my tests.  Looking through spec comments, it's required for at least the iSCI test so removing it will likely cause problems.